code/modules/mining/equipment/monster_organs/rush_gland.dm

```diff
@@ -0,0 +1,98 @@
+/// Health under which implanted gland will automatically activate
+#define HEALTH_DANGER_ZONE 30
+
+/**
+ * On use in hand, makes you run really fast for 5 seconds and ignore injury movement decrease.
+ * On use when implanted, run for longer and ignore all negative movement. Automatically triggers if health is low (to escape).
+ */
+/obj/item/organ/internal/monster_core/rush_gland
+	name = "rush gland"
+	icon_state = "lobster_gland"
+	icon_state_preserved = "lobster_gland_stable"
+	icon_state_inert = "lobster_gland_decayed"
+	desc = "A lobstrosity's engorged adrenal gland. You can squeeze it to get a rush of energy on demand."
+	desc_preserved = "A lobstrosity's engorged adrenal gland. It is preserved, allowing you to use it for a burst of speed whenever you need it."
+	desc_inert = "A lobstrosity's adrenal gland. It is all shrivelled up."
+	user_status = /datum/status_effect/lobster_rush
+	actions_types = list(/datum/action/cooldown/monster_core_action/adrenal_boost)
+
+/obj/item/organ/internal/monster_core/rush_gland/on_life(delta_time, times_fired)
+	. = ..()
+	if (owner.health <= HEALTH_DANGER_ZONE)
+		trigger_interal_action()
+
+/obj/item/organ/internal/monster_core/rush_gland/trigger_interal_action()
```

Comment:
internal?

Reply:
The proc name is meant to differentiate "thing it does if you squeeze it" vs "thing it does if you have it inside your body".
But maybe "organ action" or something might be clearer.